azure-iot-hub/azure/iot/hub/digital_twin_client.py

```diff
@@ -3,8 +3,8 @@
 # Licensed under the MIT License. See License.txt in the project root for
 # license information.
 # --------------------------------------------------------------------------
-from .auth import ConnectionStringAuthentication, AzureIdentityCredentialAdapter
-from .protocol.iot_hub_gateway_service_ap_is import IotHubGatewayServiceAPIs as protocol_client
+from . import auth as hub_auth
+from .protocol import iot_hub_gateway_service_ap_is as protocol_client
```

Comment:
This actually changes the semantics of the import. I probably wouldn't change the way IotHubGatewayServiceAPIs is imported. And if I had to, I probably wouldn't keep rolling with "protocol_client" as an alias, as you are now importing a module rather than a class